src/test/kotlin/com/terraformation/backend/accelerator/db/ReportStoreTest.kt

```diff
@@ -1568,6 +1145,125 @@ class ReportStoreTest : DatabaseTest(), RunsAsDatabaseUser {
     }
   }
 
+  @Nested
+  inner class RefreshSystemMetricValues {
+    @Test
+    fun `throws exception if no permission to review report`() {
+      @Test
+      fun `throws Access Denied Exception for non-TFExpert users`() {
```

Comment:
Not sure if nested test is needed if only one test is in the nest, but that might just be a preference thing